### PR TITLE
modified dependency boundaries

### PR DESCRIPTION
**Pull request by nek0**
As  you suggested i forked the repo and modified the boundaries in the cabal file.

### CODE REVIEW

File: yesod-markdown.cabal [hunk 1]
```diff
@@ -22,8 +22,8 @@ library
                , blaze-markup    >= 0.5   && < 0.8
                , xss-sanitize    >= 0.3.1 && < 0.4
                , directory
-               , yesod-core      >= 1.2   && < 1.3
-               , yesod-form      >= 1.3   && < 1.4
+               , yesod-core      >= 1.2   && < 1.4.7.1
+               , yesod-form      >= 1.3   && < 1.4.3.1
```
Comment (pbrisbin) [inline yesod-markdown.cabal]: I'm test-compiling now with these both as `&& < 1.5`

**Comment by pbrisbin** [comment]
OK, that worked with the 1.5 upper bound. Did you want to update this PR yourself or are you OK if I force-push over it? I'm not sure what the Travis error is about, but I may just ignore it for now.

**Comment by nek0** [comment]
Just push over it.
I'm also unsure, what this travis error is about. some kind of build testing...

**Comment by pbrisbin** [comment]
Merged as 0f1d75c and released as 0.9.3. Thanks!

**Pull request closed without merging**